chunky/src/java/se/llbit/math/PackedOctree.java

```diff
@@ -235,8 +230,7 @@ private void mergeNode(int nodeIndex, int typeNegation, int data) {
   private boolean nodeEquals(int firstNodeIndex, int secondNodeIndex) {
     boolean firstIsBranch = treeData[firstNodeIndex] > 0;
     boolean secondIsBranch = treeData[secondNodeIndex] > 0;
-    return ((firstIsBranch && secondIsBranch) || treeData[firstNodeIndex] == treeData[secondNodeIndex]) // compare types
-            && treeData[firstNodeIndex + 1] == treeData[secondNodeIndex + 1]; // compare data
+    return ((firstIsBranch && secondIsBranch) || treeData[firstNodeIndex] == treeData[secondNodeIndex]); // compare types
     // FIXME possible bug here as we always compare the data even when dealing with nodes that don't really have data
     // The data int could potentially contain some junk leftover of a previous node
     // In theory it should be reset to 0 but we need to be careful
```

Comment:
I left this comment while initially implementing the packed octree, turns out that was not a problem but now the comment is more irrelevant than ever